editor-setup/README.md

```diff
@@ -0,0 +1,8 @@
+# Editor Setup
```

Comment:
Hmm we should talk about editor stuff. Lets take this part out of this PR but hold onto it for a future PR.